docs/src/installation_instructions.md

```diff
@@ -18,13 +18,14 @@ a separate and easily reproducible environment (we suggest that users read [this
 to get a better understanding of working with environments). Best practices when starting a new Oceananigans 
 project is to 
 
-1. Start with a empty directory (in this example called `new_project`).
-2. Inside the empty directory, start Julia with the command `julia --project`.
-3. Access the built-in package manager by pressing `]` in the Julia command prompt.
-4. Add the Oceananigans package with `add Oceananigans`.
-5. Instantiate/build all dependencies `instantiate`.
-
-To do so, open a Julia REPL from the terminal and then start the package manager by hitting `]`. Then:
+1. Start with a empty directory (in this example called `new_project`)
+2. Inside the empty directory, start Julia with the command `julia --project`
+3. Access the built-in package manager by pressing `]` in the Julia command prompt
+4. Activate the new environment with the command `activate .`
```

Comment:
if you open Julia with `julia --project` it activates the project of the current directory. Thus the `activate` step is redundant.

Reply:
@navidcy @tomchor I translated these instructions into https://github.com/CliMA/Oceananigans.jl/wiki/Productive-Oceananigans-workflows-and-Julia-environments

let me know what you think.

Reply:
> if you open Julia with `julia --project` it activates the project of the current directory. Thus the `activate` step is redundant.

Not if you don't already have a Project/Manifest there. At least not in 1.6 for me:

```bash
~/Dropbox/new_project $ ls -ltrh                                                                                                                                        ✔  base  
total 0
~/Dropbox/new_project $ julia  --project                                                                                                                                ✔  base  
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.6.3 (2021-09-23)
 _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
|__/                   |

(@v1.6) pkg> activate .
  Activating new environment at `~/Dropbox/new_project/Project.toml`

(new_project) pkg> 
```

Reply:
Then you don't need to use `julia --project`:

```
gregorywagner:new_project/ $ julia                                                                [9:03:50]
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.6.5 (2021-12-19)
 _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
|__/                   |

(@v1.6) pkg> activate .
  Activating new environment at `~/Desktop/new_project/Project.toml`

(new_project) pkg>
```

Reply:
Agreed. My main reason for putting that there anyway was to get the user used to issuing `julia --project` instead of `julia`. Since it makes no difference I opted for consistency (since best practices are to use `julia --project` most times anyway). But feel free to chnge that if you disagree

Reply:
I feel recommending that users read the Pkg documentation carefully and decide for themselves when / when not to use environments is a good choice. I prefer "productive workflows" to "best practices" because the latter makes it sound like I actually know best --- which I probably don't! The former is more like "well, just trying to help..."

Reply:
What do you think about this page? 

https://github.com/CliMA/Oceananigans.jl/wiki/Productive-Oceananigans-workflows-and-Julia-environments